src/ModalHeader.js

```diff
@@ -65,9 +65,9 @@ class ModalHeader extends React.Component {
           <button
             type="button"
             className="close"
-            aria-label={label}
             onClick={createChainedFunction(modal && modal.onHide, onHide)}
           >
+            <span className="sr-only">{closeLabel}</span>
```

Comment:
order shouldn't matter but i can change it. I do want to fix the naming, i forgot this was going into `next` 👍